example/README.md

```diff
@@ -1,73 +1,43 @@
-# plaid-react-native-demo-app
+# Example App
 
-Sample apps using the React Native Plaid Link SDK on both iOS and Android.
+Our example app provides a minimal React Native app that implements Plaid Link. This app allows you to link a sample bank account.
 
-## Local Developement of react-native-plaid-link-sdk
+In order to test Plaid Link using this example app you'll need the following:
 
-To test local changes to the `react-native-plaid-link-sdk` package...
+- A [Plaid Account](https://dashboard.plaid.com/signup) to get a API keys.
+- A [Link Token](https://plaid.com/docs/api/tokens/#linktokencreate) you can quickly fetch one using your API keys and our [Postman collection](https://plaid.com/docs/api/postman/).
```

Comment:
It doesn't really matter, but for Postman https://github.com/plaid/plaid-postman is a better destination to link to than the docs. (The only reason there's an article in the docs is basically for discoverability purposes, all the content is in the GitHub.)